spec/controllers/general_controller_spec.rb

```diff
@@ -8,13 +8,22 @@
 
     before do
       expect(Statistics::General).to receive(:new).and_return(mock_stats)
+      expect(mock_stats).to receive(:to_json).with(kind_of(Hash))
```

Comment:
with `kind_of(Hash)` is this how Rails calls the method when rendering?

Reply:
Yeah – added to commit message in 84207a3 to make this a bit more obvious.